### ISSUE TITLE
is this mod still being developed?

### ISSUE THREAD

**Issue by anthony1204** [VideoGameSmash12/CFX#12]
is this mod still being developed?

**Comment by ThePlainBagel** [comment]
i hope it is id love to see a 1.21.1 version
ive yet to find a alternative to fix the many translation based crashes ive encounterd

**Comment by VideoGameSmash12** [comment]
I'm giving it another try, and I've seen some successes.

**Comment by VideoGameSmash12** [comment]
Yes, this is me metaphorically kicking the door down. I'm resuming development of the mod now that I've cleared a major obstacle and I'm aiming for a release of v1.9 hopefully by the end of the week.

This update will support the versions it supported before but also include 1.20.5+ into the mix. All that needs to be done is some testing and it should be ready to go.

**Comment by VideoGameSmash12** [comment]
8 hours later, v1.9 is now available.

[https://github.com/VideoGameSmash12/CFX/releases/tag/v1.9](https://github.com/VideoGameSmash12/CFX/releases/tag/v1.9)
